core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/apis/twitter/TwitterSourceOpExecConfig.scala

```diff
@@ -0,0 +1,15 @@
+package edu.uci.ics.texera.workflow.operators.source.apis.twitter
+import edu.uci.ics.amber.engine.architecture.breakpoint.globalbreakpoint.GlobalBreakpoint
+import edu.uci.ics.amber.engine.common.virtualidentity.{ActorVirtualIdentity, OperatorIdentity}
+import edu.uci.ics.amber.engine.operators.OpExecConfig
+
+class TwitterSourceOpExecConfig(
```

Comment:
Why do we need this class?

Reply:
since I assume all twitter related source operators can share the common config. I am only supplying different executors.